addons/sale_project/models/project.py

```diff
@@ -517,7 +552,7 @@ def _add_invoice_items(self, domain, profitability_items, with_action=True):
         )[0][0]
         revenue_items_from_invoices = self._get_revenues_items_from_invoices(
             excluded_move_line_ids=sale_lines.invoice_lines.ids,
-            with_action=with_action
+            with_action=with_action,
```

Comment:
Unnecessary diff (it is to avoid to change the last commit in that line just for that change)

Reply:
IMO, we usually blame the code in the business and not in the test.

But indeed, if we want to do it everywhere, we should apply it in the 2 lines you pointed in the test. 🙂 

It is up to you, it is not really mandatory. 🙂